charts/bulk-scan-processor/values.yaml

```diff
@@ -45,6 +45,7 @@ java:
         - processed-envelopes-queue-listen-connection-string
         - storage-account-name
         - storage-account-primary-key
+        - AppInsightsInstrumentationKey
```

Comment:
see, hyphens would be so nice :) (mentioned in the pr review linked in description here)